portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsParagraphField.testcase

```diff
@@ -32,13 +32,34 @@ definition {
 	}
 
 	@description = "Verify that a Paragraph Field can be created"
-	@ignore = "true"
 	@priority = "5"
 	test CanBeCreated {
 		property portal.acceptance = "false";
```

Comment:
Can you please add `property forms.functional = "true";` here following alphabetic order?

I mean, something like:
```
property forms.functional = "true";
property portal.acceptance = "false";
```

Thanks.